test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
```

Comment:
Not too sure if I should render the entire `homePage` here or just the `blogSection`. Since this entire file is about testing the `blogSection` I chose to render the latter.   :eyes:  :no_mouth: